web_src/css/admin.css

```diff
@@ -46,32 +47,37 @@
 .admin dl.admin-dl-horizontal {
   padding: 20px;
   margin: 0;
+  display: flex;
+  flex-wrap: wrap;
 }
 
 .admin dl.admin-dl-horizontal dd {
-  margin-left: 275px;
+  margin-left: auto;
+  width: calc(100% - 245px);
```

Comment:
can we just use `flex: 1` instead of `calc`?

Reply:
Assuming no, because the HTML structure is not nested, and just a flat list of elements, so the wrapping relies on the explicit widths to work.

Reply:
@wxiaoguang is it okay? I don't see another way given that HTML structure.

Reply:
Just a question, so LGTM